src/components/map/layers/TrackedEntityLayer.js

```diff
@@ -27,27 +27,27 @@ const getCentroid = (points) => {
 
 const fetchTEI = async (id, fieldsString, baseUrl) => {
     const data = await apiFetchWithBaseUrl({
-        url: `/trackedEntityInstances/${id}?fields=${fieldsString}`,
+        url: `/tracker/trackedEntities/${id}?fields=${fieldsString}`,
         baseUrl,
     })
     return data
 }
 
-const geomToCentroid = (type, coords) => {
-    switch (type) {
-        case 'POINT':
-            return JSON.parse(coords)
-        case 'POLYGON':
+const geomToCentroid = (geometry) => {
+    switch (geometry.type) {
+        case 'Point':
```

Comment:
Here we can also use the constants from util/geojson

Reply:
Addressed here: https://github.com/dhis2/maps-app/pull/3346/commits/fb59ae7f2f693b09c4312726e64b022965d8e00b